File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/IssueList/IssueListDepartments.pas
unit IssueListDepartments;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms,
  Dialogs, IssueListForm, ImgList, ComCtrls, ToolWin, ExtCtrls, StdCtrls,
  cxControls, cxContainer, cxEdit, cxTextEdit, cxDBEdit, DBCtrls, cxGraphics,
  cxLookAndFeels, cxLookAndFeelPainters, cxNavigator, cxLabel;

type
  TfrmDepartments = class(TfrmBasic)
    cxDBTextEdit1: TcxDBTextEdit;
    Label1: TcxLabel;
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  frmDepartments: TfrmDepartments;

implementation

{$R *.dfm}

uses
  IssueListForms, IssueListConst;

initialization

  FormManager.RegisterForm(DepartmentsFormID, TfrmDepartments, 'DEPARTMENTS');


end.

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/IssueList/IssueListDepartments.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IssueListDepartments.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "IssueListForm"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxDBEdit"
#pragma link "cxEdit"
#pragma link "cxTextEdit"
#pragma link "cxGraphics"
#pragma link "cxLabel"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxLookAndFeels"
#pragma link "cxNavigator"
#pragma resource "*.dfm"
TfrmDepartments *frmDepartments;
//---------------------------------------------------------------------------
__fastcall TfrmDepartments::TfrmDepartments(TComponent* Owner)
  : TfrmBasic(Owner)
{
}
//---------------------------------------------------------------------------
